src/taxonomy/api/types.mjs

```diff
@@ -27,6 +27,13 @@
  * @property {TaxonomyListData} data
  */
 
+/**
+ * @typedef {Object} ExportRequestParams
+ * @property {number} pk
+ * @property {string} format
+ * @property {string} name
+ */
+
```

Comment:
I think we are not using this anymore
```suggestion
```

Reply:
Updated here 0be804c079df1e8964355b265cee6f02afb3335d